tests/integ/test_tf_script_mode.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2017-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import os
+import pytest
+
+import boto3
+from sagemaker.tensorflow import TensorFlow
+from six.moves.urllib.parse import urlparse
+import tests.integ as integ
+import tests.integ.timeout as timeout
+
+RESOURCE_PATH = os.path.join(os.path.dirname(__file__), '..', 'data', 'tensorflow_mnist')
+SCRIPT = os.path.join(RESOURCE_PATH, 'mnist.py')
+DISTRIBUTION_ENABLED = {'parameter_server': {'enabled': True}}
+
+
+@pytest.fixture(scope='session', params=['ml.c5.xlarge', 'ml.p2.xlarge'])
+def instance_type(request):
+    return request.param
+
+
+def test_mnist(sagemaker_session, instance_type):
```

Comment:
should we add one of these to the continuous testing suite?

Reply:
I will do that in a follow up pr. The test only works in pdx now. I need to get the container change out so the test can work in all the regions.